packages/saasify-cli/lib/schemas/config.schema.json

```diff
@@ -103,34 +101,13 @@
           "type": "object",
           "title": "input"
         },
-        "output": {
-          "title": "output",
-          "oneOf": [
-            {
-              "type": "object"
-            },
-            {
-              "type": "string"
-            },
-            {
-              "type": "array"
-            }
-          ]
-        },
         "inputContentType": {
           "type": "string",
           "title": "inputContentType"
         },
-        "outputContentType": {
-          "type": "string",
-          "title": "outputContentType"
-        },
```

Comment:
Do we want to issue a deprecation warning?

Reply:
nah; nobody's using this stuff aside from us. we'll def have to do this in the future though once we get more makers onboard.